src/Entity/IdeasWorkshop/Idea.php

```diff
@@ -706,6 +752,34 @@ public function getReportType(): string
         return ReportType::IDEAS_WORKSHOP_IDEA;
     }
 
+    /**
+     * @SymfonySerializer\Groups("idea_read")
+     */
+    public function isExtendable(): bool
+    {
+        return $this->extendsCount < self::EXTEND_COUNT_LIMIT;
```

Comment:
this will return `true` for a draft idea too, what about adding the status check here?

Reply:
I can add test here ut it will means that I will not be able to know if the 400 is from status or limitation ?